docs/usage/dashboards.md

```diff
@@ -97,15 +121,17 @@ The widget configuration will be saved as a JSON file, with the title formatted
 
 Dashboards can be imported from the custom dashboards list:
 
-1. Hover over the Add button (+) in the right bottom corner.
-2. Click on the `Import dashboard` button (cloud with an upward arrow).
-3. Select your file.
+1. Click on the `Import dashboard` button in the right top corner
+2. Select your file.
 
-![Import dashboard option](assets/import-dashboard-option.png)
+![Import dashboard option](assets/import_dashboard.png)
 
-To import a widget, the same mechanism is used, but from a dashboard view.
+To import a widget from a dashboard view,
+1- Hover over the Add button (+) in the right bottom corner.
+2- Click on the Import a widget button (cloud with an upward arrow).
+3- Select your file.
 
-![Import widget option](assets/import-widget-option.png)
+![Import widget option](assets/import_widget.png)
 
 !!! warning "Configuration compatibility"
```

Comment:
@CelineSebe was this here before? Will it appear on the page?

Reply:
This was here before, but it will not appear on the page

Reply:
This was here before, and it appeared on the page.